utils/check_dashboard_name_uniqueness.ts

```diff
@@ -23,8 +23,9 @@ interface DashboardConfigs {
 /** Returns any quickstart dashboards with matching names and their filepaths */
 const findMatchingDashboardNames = (): DashboardNamesAndPaths[] => {
   const dashboardConfigs = glob.sync(
-    path.resolve(process.cwd(), '../quickstarts/**/dashboards/*/*.+(json)')
+    path.resolve(process.cwd(), '../quickstarts/**/dashboards/*.+(json)')
```

Comment:
We want to keep this glob pattern the same as main. Merging in the main branch should get you there